### PR TITLE
yhorodechna add task "HTML-CSS-Popup"

### PR DESCRIPTION
**Pull request by yhorodechna**
###HTML-CSS-Popup

[Demo](https://yhorodechna.github.io/html-css-popup/) |
[Code base](https://github.com/yhorodechna/html-css-popup.git)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/yhorodechna/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            <div class="popup__body">
+                <input tabindex="1" class="popup__input" type="checkbox" id="popup__input" />
+                <label for="popup__input" class="popup__label  menu__item">
+                    <img src="img/header/popup-button.png" alt="popup picture" />
+                </label>
+                <nav class="popup">
+                    <ul class="popup__list">
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/calendar.png" alt="calendar picture" />
+                            <span>Calendar</span>
```
Comment (A-Ostrovnyy) [inline submissions/yhorodechna/html-css-popup/index.html:24]: Logically elements inside the popup are a list of links.

File: submissions/yhorodechna/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/message.png" alt="message picture" />
+                            <span>Message</span>
+                        </li>
+                        <li class="popup__item" tabindex="2">
+                            <img src="img/phone.png" alt="phone picture" />
+                            <span>Phone</span>
+                        </li>
+                        <li class="popup__more more">
+                            <input tabindex="2" class="more__input" type="checkbox" id="more__input" />
```
Comment (A-Ostrovnyy) [inline submissions/yhorodechna/html-css-popup/index.html:59]: By default, all interractive elements already have a focus state, adding `tabindex` attribute you change the default order of interactive elements set by HTML markup. In the current task no need to use `tabindex` attribute unless you want to change default order)

File: submissions/yhorodechna/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                    <li class="menu__item" tabindex="4"><img src="img/header/search.png" alt="search picture" />
+                    </li>
+                    <li class="menu__item" tabindex="4"><img src="img/header/notification.png"
+                            alt="notification picture" /></li>
+                    <li class="menu__item" tabindex="4"><img src="img/header/user.png" alt="user picture" /></li>
+                    <li class="menu__item" class="test" tabindex="4"><img src="img/header/settings.png"
+                            alt="settings picture" />
+                    </li>
+                </ul>
+            </div>
```
Comment (A-Ostrovnyy) [inline submissions/yhorodechna/html-css-popup/index.html:102]: If this block a menu why you don't use `nav` tag?

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

File: submissions/yhorodechna/html-css-popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  justify-content: center;
+  align-content: center;
+}
+.popup__item {
+  width: 70px;
+  height: 70px;
+  padding: 5px;
+  margin: 5px;
+}
+span.more__btn {
```
[the remark this answers is not in the document]
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/css/style.css]: Thanks , I solved this problem.

File: submissions/yhorodechna/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        </header>
+        <main class="main">
+        </main>
+        <footer class="footer">
+        </footer>
```
[the remark this answers is not in the document]
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html]: Deleted all unused tags.

File: submissions/yhorodechna/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    </div>
+</body>
+
+</html>
+
```
[the remark this answers is not in the document]
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html:135]: Deleted all empty lines. About empty line at the end of each file I keep in mind)

File: submissions/yhorodechna/html-css-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    <meta name="viewport" content="width=device-width , initial-scale=1.0" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="popup__body">
+                <input tabindex="1" class="popup__input" type="checkbox" id="popup__input" />
+                <label for="popup__input" class="popup__label  menu__item">
+                    <img src="img/header/popup-button.png" alt="popup picture" />
```
[the remark this answers is not in the document]
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html]: Thanks , I solved this problem.

File: submissions/yhorodechna/html-css-popup/index.html (hunk 1 above)
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html]: Earlier I used links , but decided to deleted them . Now returned them.

File: submissions/yhorodechna/html-css-popup/index.html (hunk 2 above)
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html]: Deleted all tabindex.

File: submissions/yhorodechna/html-css-popup/index.html (hunk 3 above)
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html]: I missed it . Solved.

File: submissions/yhorodechna/html-css-popup/index.html [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+                        <li class="popup__item">
+                            <a href="#" class="popup__link">
+                                <img class="img" src="img/phone.png" alt="" />
+                                <span class="subtitle">Phone</span>
+                            </a>
+                        </li>
+                        <li class="popup__more more">
+                            <input class="more__input" type="checkbox" id="more__input" />
+                            <label class="more__label" for="more__input"><span class="subtitle more__btn">
+                                </span></label>
```
Comment (A-Ostrovnyy) [inline submissions/yhorodechna/html-css-popup/index.html:77]: When the second part of the popup is open your button looks not ok) (not 100% width) also on hover and focus button background overlaps the bottom border of the popup. Also don't forget about border-radius:button more doesn't have it and popup has)
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Awesome!
Couple issues need to fix)

File: submissions/yhorodechna/html-css-popup/index.html [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+                                    </a>
+                                </li>
+                            </ul>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <nav class="menu">
+                <ul>
+                    <li class="menu__item"><img class="img" src="img/header/search.png" alt="" />
```
[the remark this answers is not in the document]
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html]: I forgot about that. Solved.

File: submissions/yhorodechna/html-css-popup/index.html (hunk 8 above)
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html:77]: It was my big mistake... 
Fixed.

File: submissions/yhorodechna/html-css-popup/index.html [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+            </div>
+            <nav class="menu">
+                <ul class="menu__list">
+                    <li><a class="menu__item" href="#"><img class="img" src="img/header/search.png" alt="" /></a>
+                    </li>
+                    <li><a class="menu__item" href=""><img class="img" src="img/header/notification.png" alt="" /></a>
+                    </li>
+                    <li><a class="menu__item" href=""><img class="img" src="img/header/user.png" alt="" /></li></a>
+                    <li><a class="menu__item" href=""><img class="img" src="img/header/settings.png" alt="" /></a>
+                    </li>
```
Comment (A-Ostrovnyy) [inline submissions/yhorodechna/html-css-popup/index.html:128]: For the future don't forget about consistency: if you use `#` as a value for `href` attribute you should use it for all empty `href` attributes)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

File: submissions/yhorodechna/html-css-popup/index.html (hunk 10 above)
Comment (yhorodechna) [reply submissions/yhorodechna/html-css-popup/index.html:128]: Ok, thanks)

**Pull request merged**